### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/faq/troubleshooting.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#968] [Announcements]
# https://theme-hope.vuejs.press/zh/faq/troubleshooting.html

确认使用正确的环境 vuepress-theme-hope 仅支持 >=18.19.0 版本的 Node.js。 你可以通过 node -v 查看它的版本，如果不满足要求，请 。 同时，你应该使用满足下方要求的包管理器 (推荐 pnpm@9): pnpm: >= 7 检查: pnpm -v npm: >= 8 检查: npm -v yarn: >= 2...

https://theme-hope.vuejs.press/zh/faq/troubleshooting.html

**Comment by jannchan** [comment]
22
